acceptance/lib/helper.rb

```diff
@@ -25,7 +25,7 @@ def self.initialize_config(options)
 
     puppet_version = get_option_value(options[:puppet_version],
                          nil, "Puppet Version", "PUPPET_VERSION",
-                         "4.99.0.31.g17aa7d8",
+                         "4.99.0-125-gf9a1b77",
```

Comment:
Ah, I probably should use the dots instead good catch.  I think I got that from running `git describe` on the commit.  Must be some logic which subs hyphens for dots somewhere.  The `SUITE_VERSION` on the puppet-agent job uses the dot form, as does the name of the rpm file on the build server - http://builds.delivery.puppetlabs.net/puppet-agent/f9a1b779d71164f7325bd5cdd63077aa20e4cd29/artifacts/el/7/PC1/x86_64/puppet-agent-4.99.0.125.gf9a1b77-1.el7.x86_64.rpm - so that seems like what we should use.  I'll fix that up.